public/video-ui/src/services/WorkflowApi.js

```diff
@@ -149,7 +149,21 @@ export default class WorkflowApi {
     });
   }
 
-  static updateStatus({id, status}) {
+  static updateProdOffice({ id, prodOffice }) {
+    const payload = {
+      data: prodOffice
+    };
+
+    return pandaReqwest({
+      method: 'PUT',
+      url: `${WorkflowApi.workflowUrl}/api/stubs/${id}/prodOffice`,
```

Comment:
This doesn't appear to be a valid route in Workflow.

Reply:
Ah, it is, it's just blocking the request due to CORS. We'd want to enable it first (e.g https://github.com/guardian/workflow-frontend/pull/170)

Reply:
ah great point thank you. I assumed the CORS flagged up because of an environment issue. Will confirm on code and merge